src/components/request/Request.tsx

```diff
@@ -2,10 +2,18 @@ import { useEffect } from "react";
 import Table from "../table/Table";
 import { useSearchParams } from "react-router-dom";
 import useRequestTableData from "./useRequestTableData";
-import { Wrapper, TableContentWrapper, TableSection } from "./Request.styled";
+import {
+  Wrapper,
+  TableContentWrapper,
+  TableSection,
+  TableTitle,
+} from "./Request.styled";
 import RequestHero from "./RequestHero";
 import useClient from "hooks/useOracleClient";
-
+import useReader from "hooks/useOracleReader";
+import { oracle } from "@uma/sdk";
```

Comment:
I dont think u ever need to import this, you can get it from the useoracleclient hook

Reply:
Changed!